.github/workflows/ruff-check.yml

```
@@ -13,52 +13,66 @@ permissions:
 jobs:
   ruff-changed-files:
     runs-on: ubuntu-latest
-
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)